qrand.go

```diff
@@ -66,6 +68,25 @@ func Read(buf []byte) (n int, err error) {
 	return Reader.Read(buf)
 }
 
+type ΨSource struct{}
```

Comment:
Although I'm fond of the name `ΨSource`, because I'm a physicist, I have to reluctantly accept that it won't mean much to normal people. In general, if possible we should follow the same conventions and names as the standard library, so shall we name this `Source` instead?